packages/lisk-transactions/src/1_register_second_passphrase.js

```diff
@@ -16,7 +16,15 @@ import cryptography from 'lisk-cryptography';
 import { SIGNATURE_FEE } from './constants';
 import { wrapTransactionCreator } from './utils';
 
-const registerSecondPassphrase = ({ secondPassphrase }) => {
+const validateInputs = ({ secondPassphrase }) => {
+	if (typeof secondPassphrase !== 'string') {
+		throw new Error('Please provide a secondPassphrase. Expected string.');
```

Comment:
Should we prefix every such error message with `Could not create transaction: `? It would be more consistent.

Reply:
@willclarktech actually only transfer tx creator has `Could not create transaction: `. All the other error messages don't have this prefix. I'd prefer to remove it since we will get it during transfer creation anyway.